app/forms/field_handlers/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from app.questionnaire.value_source_resolver import ValueSourceResolver
 
+from ...questionnaire.rules.rule_evaluator import RuleEvaluator
```

Comment:
Any reason not to use absolute import here?

Reply:
Think IDE auto imported following the rest below, which are all relative. Have changed all to be absolute.